scripts/subscribe.py

```diff
@@ -31,16 +31,16 @@ def main(argv):
             f.write(creds['SecretString'])
             f.flush()
             with patch.dict(os.environ, GOOGLE_APPLICATION_CREDENTIALS=f.name):
-                subscribe(options, dss_client)
+                subscribe(options.subscribe, dss_client)
```

Comment:
The arguments are in the a different order because I changed the arguments of the subscribe() method below.

Reply:
I am fully aware why you changed the signature. But when you do such a refactoring you need to make sure that you 1) refactor all call sites and 2) test all call sites, either manually or automatically by writing tests. When you say:

> The arguments are in the a different order because I changed the arguments of the subscribe() method below.

that might be interpreted as though you think my criticism was unjustified. It wasn't. Just making sure we're on the same page. You did apply my suggested change in a subsequent push which indicates that we are.